squashfs/index.html

```diff
@@ -1156,7 +1155,7 @@ <h4 id="directory-index">Directory Index</h4>
                             <tr>
                                 <td>index</td>
                                 <td>u32</td>
-                                <td>An offset into the uncompressed metadata block</td>
+                                <td>This stores a byte offset from the first directory header to the current header, as if the uncompressed directory meta data blocks were laid out in memory consecutively.</td>
```

Comment:
Elsewhere we use metadata as a single word, and I think this use should be as well.

Reply:
Fixed it in the latest commit